ejviola/mailroom.py

```diff
@@ -22,23 +23,31 @@
         if row['Cause'] not in cause_list:
             cause_list.append(row['Cause'])
     report = []
-except:
-    print("File could not be opened, check: " + str(donor_data_file_name))
+
+except (EOFError, FileNotFoundError) as e:
+    with open(donor_data_file_name, 'wb') as donor_data_file:
+        pickle.dump([{'Name': "Bill Gates", 'Amount': 115000000, 'Date': "August 1, 1991",
+                                    'Cause': 'combat homelessness.'}, {'Name': "Bill Gates", 'Amount': 9176200, 'Date': "January 3, 1999",
+                                    'Cause': "support global literacy."}, {'Name': "Bill Gates", 'Amount': 115000000, 'Date': "September 1, 1999",
+                                    'Cause': "engage in local outreach efforts to raise awareness about homelessness."}, {'Name': "Paul Allen", 'Amount': 3150000, 'Date': "November 12, 2000",
+                                    'Cause': "support global literacy."}, {'Name': "Bill Gates", 'Amount': 115000000, 'Date': "August 17, 2001",
+                                    'Cause': "combat homelessness."}], donor_data_file)
+    print(e)
+    print("Wrote 5 starter records to " + donor_data_file_name + " and successfully exited.")
     sys.exit()
 
+
```

Comment:
I would suggest one full item per line instead of all the items broken in the middle:
```
pickle.dump([{'Name': "Bill Gates", 'Amount': 115000000, 'Date': "August 1, 1991",'Cause': 'combat homelessness.'},
			 {'Name': "Bill Gates", 'Amount': 9176200, 'Date': "January 3, 1999", 'Cause': "support global literacy."},
			 {'Name': "Bill Gates", 'Amount': 115000000, 'Date': "September 1, 1999",'Cause': "engage in local outreach efforts to raise awareness about homelessness."},
			 {'Name': "Paul Allen", 'Amount': 3150000, 'Date': "November 12, 2000", 'Cause': "support global literacy."},
			 {'Name': "Bill Gates", 'Amount': 115000000, 'Date': "August 17, 2001",'Cause': "combat homelessness."}], donor_data_file)
```